app/models/podcast.rb

```diff
@@ -14,8 +14,18 @@ class Podcast < BaseModel
 
   acts_as_paranoid
 
+  before_validation :set_defaults
+
   scope :published, -> { where('published_at IS NOT NULL AND published_at <= now()') }
 
+  def set_defaults
+    self.enclosure_template ||= enclosure_template_default
+  end
+
+  def enclosure_template_default
+    "https://#{ENV['DOVETAIL_HOST']}/{slug}/{guid}/{original_filename}"
```

Comment:
Is the assumption that all of feeder will go through Dovetail then?  Rather than allowing someone to use the raw `f.prxu.org` enclosures?  (Or maybe I'm misreading this entirely).